### PR TITLE
Add food

### PR DESCRIPTION
**Pull request by artyomtrityak**

### CODE REVIEW

File: economy.cpp [hunk 1]
```diff
@@ -765,15 +765,15 @@ void ARegion::SetupProds(double weight)
 			switch (foodchoice) {
 				case 0:
 					if (!(ItemDefs[I_GRAIN].flags & ItemType::DISABLED))
-						p = new Production(I_GRAIN, typer->economy * 2);
```
Comment (artyomtrityak) [inline economy.cpp:768]: it was added to increase food but not needed anymore

File: gamedata.cpp [hunk 2]
```diff
@@ -1731,7 +1731,7 @@ ItemType id[] =
 	 ItemType::DISABLED,
 	 "WEAP",3,1,1, {{I_IRONWOOD,1},{-1,0},{-1,0},{-1,0}},
 	 NULL,0,0, {{-1,0},{-1,0},{-1,0},{-1,0}},
-	 2, IT_ADVANCED|IT_WEAPON, 90, 1,
```
Comment (artyomtrityak) [inline gamedata.cpp:1734]: new battle axe cost for drops

File: gamedata.cpp [hunk 3]
```diff
@@ -4197,7 +4197,7 @@ static TerrainType td[] = {
 	{"ocean", "ocean", '-', R_OCEAN,
 	 TerrainType::BARREN | TerrainType::FLYINGMOUNTS | TerrainType::SHOW_RULES,
 	 0,0,0,1,
-	 {{I_FISH,100,40},{I_TURT,30,10},{I_ROOTSTONE,5,5},{-1,0,0},
```
Comment (artyomtrityak) [inline gamedata.cpp:4200]: no ROOT in ocean

File: neworigins/extra.cpp [hunk 4]
```diff
@@ -881,6 +881,10 @@ void Game::ModifyTablesPerRuleset(void)
 
 	DisableSkill(S_GEMCUTTING);
 
+	// Food
+	EnableSkill(S_COOKING);
```
Comment (artyomtrityak) [inline neworigins/extra.cpp:885]: Enable food

**Review by valdisz — APPROVED** [review]

**Pull request merged**